src/style-spec/reference/v8.json

```diff
@@ -4651,6 +4651,50 @@
         ]
       },
       "property-type": "data-constant"
+    },
+    "fill-extrusion-ambient-occlusion-intensity": {
+      "property-type": "data-constant",
+      "type": "number",
+      "default": 0.0,
+      "minimum": 0,
+      "maximum": 1,
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "zoom"
+        ]
+      },
+      "transition": true,
+      "doc": "Controls the intensity of ambient occlusion (AO) shading. Current AO implementation is a low-cost best-effort approach that shades area near ground and concave angles between walls. Default value 0.0 disables ambient occlusion and values around 0.5 provide the most plausible results for buildings.",
+      "sdk-support": {
+        "basic functionality": {
+          "js": "2.8.3",
+          "android": "10.7.0",
+          "ios": "10.7.0"
+        }
+      }
+    },
+    "fill-extrusion-ambient-occlusion-radius": {
+      "property-type": "data-constant",
+      "type": "number",
+      "default": 3.0,
+      "minimum": 0,
+      "maximum": 1000000,
```

Comment:
Do we even need a maximum bound at all instead of a such a high value? I'm not sure if providing a `minimum` without a `maximum` value is valid from the spec though.

Reply:
Removed max, works fine without it.